ci/shared/scripts/deps/mesa.sh

```diff
@@ -21,12 +22,15 @@ if [ "$OS" == "Ubuntu" ]; then
         apt-get update
     fi
 
+    apt-get update -y -q
     apt-get install -y -q --no-install-recommends \
         libgl1-mesa-dev \
         libglu1-mesa-dev \
         mesa-common-dev \
         libosmesa6-dev \
         mesa-utils
+    apt-get clean -y -q
+    rm -rf /var/lib/apt/lists/*
```

Comment:
We normally clean-up right after installing the dependencies, so we have to repeat `update` and `clean` here. This should be fine considering the whole mesa script is a workaround for Ubuntu 24.04 issue. Hopefully this gets moved to `extra_packages` in docker-bake.hcl` eventually.